netbout-web/src/main/java/com/netbout/cached/CdMessages.java

```diff
@@ -146,6 +146,7 @@ public long unread() throws IOException {
         }
         /**
          * I've seen them all.
+         * @checkstyle NonStaticMethodCheck (5 lines)
          */
         @Cacheable.FlushBefore
         public void touch() {
```

Comment:
@mkordas I am not sure the method is supposed to be static. If yes, no problem to make it static then.